tests/test_secret.py

```diff
@@ -15,14 +15,19 @@
 from __future__ import absolute_import, division, print_function
 
 import binascii
+import re
+
+from hypothesis import given, strategies as st
 
 import pytest
 
 from utils import flip_byte
 
 from nacl.encoding import HexEncoder
 from nacl.exceptions import CryptoError
-from nacl.secret import SecretBox
+from nacl.secret import Aead, SecretBox
+
+from test_aead import xchacha20poly1305_ietf_vectors
```

Comment:
`from .test_aead import xchacha20poly1305_ietf_vectors` should work without requiring changes to application-import-names

Reply:
I tried that first, and got `ImportError: attempted relative import with no known parent package`.

Reply:
Oh we need to add an empty `__init__.py` in the tests dir so that it's treated as a package.

Reply:
I didn't suggest that, as it involved changing all the test files (`from utils import ...` had to be changed to `.utils` and moved to the “first party” import group) but it's now done.